.github/workflows/update_dev_version.yml

```
@@ -4,6 +4,11 @@ on:
     branches:
       - master
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Temporarily changing this to false for checking CI jobs:
```suggestion
  cancel-in-progress: false
```

---

Any reason for this @maliasadi ?